// Generate base consts
const bool aBool = true;
const byte aByte = 1;
const i16 a16BitInt = 12;
const i32 a32BitInt = 123;
const i64 a64BitInt = 1234;
const float aFloat = 0.1;
const double aDouble = 0.12;
const string aString = "Joe Doe";
